examples/Example/src/fling/index.tsx

```diff
@@ -22,18 +26,25 @@ class Fling extends Component {
     this._translateY = new Animated.Value(0);
   }
 
-  _onHorizontalFlingHandlerStateChange = ({ nativeEvent }, offset) => {
+  _onHorizontalFlingHandlerStateChange = (
+    {nativeEvent}: FlingGestureHandlerStateChangeEvent,
+    offset: number
+  ) => {
     if (nativeEvent.oldState === State.ACTIVE) {
       Animated.spring(this._touchX, {
+        // @ts-ignore private property
         toValue: this._touchX._value + offset,
```

Comment:
This can be probably replaced with [`Animated.add`](https://reactnative.dev/docs/animated#add).

Reply:
`toValue` does not accept `Animated.AnimatedAddition`.

Reply:
Oh well, let's leave it as it is.